### PR TITLE
ncm-opennebula: config template cleanup

### PR DESCRIPTION
**Pull request by alvarosimon**
Just a template cleanup

### CODE REVIEW

File: ncm-opennebula/src/main/pan/components/opennebula/config.pan [hunk 1]
```diff
@@ -5,6 +5,20 @@
...
+
+# Set prefix to root of component configuration.
+prefix '/software/components/${project.artifactId}';
+'active' ?= true;
+'dispatch' ?= true;
+
+
+# Package to install
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+'/software/components/${project.artifactId}/dependencies/pre' ?= list('spma', 'accounts', 'sudo', 'useraccess');
```
Comment (stdweird) [inline ncm-opennebula/src/main/pan/components/opennebula/config.pan]: move this above and use the prefix-ed path

**Comment by stdweird** [comment]
specify that this is template cleanup

**Comment by stdweird** [comment]
squash the commits in one

File: ncm-opennebula/src/main/pan/components/opennebula/config.pan [hunk 2]
```diff
@@ -5,6 +5,19 @@
...
+prefix '/software/components/${project.artifactId}';
+'active' ?= true;
+'dispatch' ?= true;
+'version' ?= '${no-snapshot-version}';
+'dependencies/pre' ?= list('spma', 'accounts', 'sudo', 'useraccess');
+
+# Package to install
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+include 'components/${project.artifactId}/sudo';
```
Comment (stdweird) [inline ncm-opennebula/src/main/pan/components/opennebula/config.pan]: why is the added?

Comment (alvarosimon) [inline ncm-opennebula/src/main/pan/components/opennebula/config.pan]: That was already included in the old `config.pan` file, required to setup oneadmin sudoers conf to be able to execute some ovs commands for example.

**Pull request merged**